addons/skin.estuary/xml/Custom_1101_SettingsList.xml

```diff
@@ -60,6 +60,12 @@
 						<label2>[B]$INFO[VideoPlayer.AudioLanguage][/B]</label2>
 						<onclick>AudioNextLanguage</onclick>
 					</control>
+          <control type="button" id="11105">
```

Comment:
whitespace oops

Reply:
I thought our codingstyle prohibits tabs but all those xml files are using tabs :(

Reply:
Well for XML files it makes sense to use tabs instead of spaces because you actually save space when using tabs. For compiled code it doesn't really matter because the tabs/spaces obviously don't end up in the compiled binary but for raw XML files it does make a difference.